src/admin/shared/components/ContentPicker/helpers/parse-picker.ts

```diff
@@ -34,7 +34,7 @@ export const parseSearchQuery = (input: string): string => {
 			)
 		);
 
-		return 'Ongeldige zoekfilter';
+		return '';
```

Comment:
```suggestion
		return tText('Ongeldige zoekfilter');
```
en dan npm run extract-translations runnen

Reply:
Will not fix, hier een string met > 0 length returnen is de oorzaak van https://meemoo.atlassian.net/browse/AVO-3113

Reply:
Dan zetten we beter op regel 11:
```
// Empty search queries are also valid and do not have to be parsed
if (!input) {
	return '';
}
```

Zodat we wel nog een error krijgen bij invalid search urls?

Reply:
Bij backspace is input `Ongeldige zoekfilte` dus dan zit je in een infinite loop.